src/App.jsx

```diff
@@ -8,8 +8,7 @@ import Header from './components/Header';
 import { useState } from 'react';
 
 function App() {
-
-	const [isLoading, setIsLoading] = useState(true)
+	const [isLoading, setIsLoading] = useState(true);
```

Comment:
we should have a separate loading state for the different components as the loading state is currently affecting the both here - if both requests don't complete at the same time, one of them will show nothing as loading will be set to false before the data has come back.